samples/Dummy/README.md

```diff
@@ -0,0 +1,5 @@
+# Dummy Project
+
+This was the quickest and easiest way to stop KoreBuild from breaking. It doesn't like when there **is** a `samples` directory but **none** of the projects have `project.json` files.
```

Comment:
We never had Hello World, we had a Test Client. I can add some kind of echo server sample.